DemoData/TheBowl/TheBowl.json

```diff
@@ -5583,7 +5583,7 @@
       "localTurbineWindSpeedsOutputSettings": null
     },
     "numberOfDirectionSectorsForWakeCalculation": 180,
-    "maximumWindSpeedForEvaluation_m_per_s": 0.0,
+    "maximumWindSpeedForEvaluation_m_per_s": 30.0,
```

Comment:
Any reason why change this? Leaving 0.0 should evaluate up to the last non 0 probability